tools/patch_charm_modules.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
[I'm a newbie with shell scripts] should this have something like a `-e` so that if there is an error in the script it propagates and the script returns the error?

Reply:
Yes, there should be a `-e` somewhere, either on this line or as a `set -e`.

Because of limitations in how the #! line is parsed (by the kernel?  I don't actually know.) you only get to pass one argument to the called program.  Here that argument is "bash", so `#!/usr/bin/env bash -e` will fail (It will try to run the program `bash -e`.), but env has special handling for this and you can write `#!/usr/bin/env -S bash -e` and env will make it work.